apps/academy-wep/components/learn/learning-module-card.tsx

```
@@ -0,0 +1,177 @@
+      key={module.id}
+      initial={{ opacity: 0, y: 20 }}
+      animate={{ opacity: 1, y: 0 }}
+      exit={{ opacity: 0, y: -20 }}
+      transition={{ duration: 0.4, delay: index * 0.05 }}
+      className={`group relative overflow-hidden bg-white rounded-2xl transition-all duration-500 hover:shadow-xl flex flex-col h-full border border-gray-100 shadow-sm ${
+        !module.unlocked ? "opacity-90" : ""
+      }`}
+      aria-label={`Module card: ${module.title}`}
+    >
+      <div className="absolute top-0 right-0 w-12 h-12 bg-gray-50 rounded-bl-2xl z-10 flex items-center justify-center">
+        <div className="w-8 h-8 bg-white rounded-bl-xl shadow-sm"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)